modyn/tests/selector/internal/storage_backend/local/test_local_storage_backend.py

```diff
@@ -0,0 +1,142 @@
+import shutil
```

Comment:
Thanks for the tests here - we need to slightly adjust this depending on how we decide to change the structure: If we have the CPP Python wrapper still in a separate file, let's test that individually and then the LocalStorageBackend, which uses this wrapper, separately.

Additionally, similarily to the triggersamplestorage, it would be great to have a couple of C++ tests as well to have the sanitizer check this as well.

Reply:
Done